pkg/cli/mirror/create.go

```diff
@@ -166,14 +193,72 @@ func (o *MirrorOptions) createOlmArtifactsForOCI(ctx context.Context, cfg v1alph
 			return err
 		}
 
+		// setup where the FBC content will be written to
 		catalogContentsDir := filepath.Join(artifactsFolderName, ctlg.Ref.Name)
+		// obtain the path to where the OCI image reference resides
+		layoutPath := layout.Path(v1alpha2.TrimProtocol(ctlg.OCIFBCPath))
+
+		// get its index.json and obtain its manifest
+		rootIndex, err := layoutPath.ImageIndex()
+		if err != nil {
+			return err
+		}
+		rootIndexManifest, err := rootIndex.IndexManifest()
+		if err != nil {
+			return err
+		}
+
+		// attempt to find the first image reference in the layout...
+		// for a manifest list only search one level deep.
+		var img v1.Image
+	loop:
```

Comment:
Labels are optional for break statements, is there any specific reasons of using label (loop) here? If there is not specific reason, it is possible simplying the code removing these labels.